tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -255,7 +312,9 @@ batch:
         compute-type: BUILD_GENERAL1_LARGE
         image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-aarch:amazonlinux-2023_clang-15x_sanitizer_latest
```

Comment:
This is still using run_ssl_asan_tests.yml

Reply:
My bad, thanks for the call out